.github/actions/setup-etcd/action.yml

```diff
@@ -0,0 +1,45 @@
+name: 'Setup etcd'
+description: 'Setup tarantool environment and add it to the PATH'
+inputs:
+  etcd-version:
+    description: 'Release name from https://github.com/etcd-io/etcd/releases'
+    required: true
+    default: v3.4.13
+  install-prefix:
+    description: 'Where to extract the archive'
+    default: './etcd'
+  export-path:
+    description: 'Whether to export PATH environment variable'
+    default: false
+runs:
+  using: 'composite'
+  steps:
+    - shell: bash
+      env:
+        BASE_URL: "https://github.com/etcd-io/etcd/releases/download"
+        ETCD_VER: ${{ inputs.etcd-version }}
+        INSTALL_PREFIX: ${{ inputs.install-prefix }}
+      run: >
```

Comment:
Why do you use `<` here but `|` below? Is there any difference? If there is no difference I think it's better to uniform style

Reply:
They're different. `|` preserves newlines, while `>` replaces newlines with spaces (folds the block)

See also https://yaml-multiline.info/

Reply:
Thanks for explanation. Ok.